### PR TITLE
fix: invite accepted event

### PR DESCRIPTION
**Pull request by suprko**
#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/6764/desktop-contact-accepted-notification-snackbar-tweak

#### Testing instructions  
1. when you add or remove contacts from favourites, snackbar should no longer be displayed
2. when contact accepts invite, they shouldn't be added to favourites by default
3. PendingDM actually broke the desktop notification for accepted invites and we didn't notice. verify that these are working again. (invite new user -> new user signs up and confirms email -> original user sees desktop notification)

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: src/models/contacts/contact-store.js [hunk 1]
```diff
@@ -207,11 +207,18 @@ class ContactStore {
...
-            () => this.invites.loaded,
+            () => this.invites.loaded && tofuStore.loaded && getChatStore().loaded,
             () => {
                 try {
                     this.invitedContacts.forEach(c => {
                         if (c.username) {
+                            // If c.username exists, then invited user has indeed joined Peerio & confirmed email.
+                            // But, if same username isn't in tofuStore, current user doesn't yet know this,
+                            // so emit `onInviteAccepted` event (on desktop this shows a notification).
+                            if (!tofuStore.cache[c.username]) {
```
Comment (anri-asaturov) [inline src/models/contacts/contact-store.js]: So, the thing is, tofuStore caches a maximum of 500 contacts, so for users with more then 500 tofu kegs this won't work. Maybe add a story for later.

Comment (suprko) [reply src/models/contacts/contact-store.js]: done https://app.clubhouse.io/peerio/story/7409/sdk-revisit-invite-accepted-event-due-to-tofustore-limit

**Comment by ebilg** [comment]
Lucas could you please merge dev into your branch?

**Comment by anri-asaturov** [comment]
@ebilg did it

**Comment by ebilg** [comment]
can be merged

**Pull request merged**